src/Math-TSNE/PMTSNE.class.st

```diff
@@ -198,23 +203,23 @@ PMTSNE >> perplexityDefaultValue [
 
 { #category : #running }
 PMTSNE >> runPcaOnX [
```

Comment:
Maybe we need to change the name X and be more explicit about its meaning in the t-SNE algorihm.

Reply:
Are you talking about instance variable `x` or the method name `runPcaOnX`?
We can:
1. Change `x` to `input` and change `runPcaOnX` to `reduceInputToInputDims`
2. Or change change `runPcaOnX` to `reduceXToInputDims`

Reply:
Changed in https://github.com/PolyMathOrg/PolyMath/pull/117/commits/49f3b2a37d019b821aff18267a025b2175437a82
Is it better now?